deploy/cdk/lib/cdk-pipeline-deploy.ts

```diff
@@ -162,7 +162,7 @@ export class CDKPipelineDeploy extends Construct {
         's3:GetBucketLocation',
         's3:GetBucketPolicy',
       ],
-      resources: [ 'arn:aws:s3:::cdktoolkit-stagingbucket-*' ],
+      resources: ['arn:aws:s3:::cdktoolkit-stagingbucket-*', 'arn:aws:s3:::cdk*' ],
```

Comment:
Is the first parameter covered by the second?

Reply:
It does.  I was testing various things incrementally, and finally arrived on this that worked. I toyed with removing the first, but thought I'd wait until after we migrate this to cdk2, after which point we won't need any reference to the cdktoolkit-stagingbucket.